.github/pull_request_template.md

```diff
@@ -1,11 +1,21 @@
-# Description
+### Description
 
-> Detailed changes summary | testing | dependencies
-> Good example: https://os.mbed.com/docs/latest/reference/guidelines.html#workflow (Pull request template)
+<!-- 
+    Required
+    Add here detailed changes summary, testing results, dependencies 
+    Good example: https://os.mbed.com/docs/latest/reference/guidelines.html#workflow (Pull request template)
+-->
 
-# Pull request type
+
+### Pull request type
+
+<!-- 
+    Required
+    Please tick one of the following types 
```

Comment:
Wit the additions of _New target_ and _Breaking change_ should this read "Please tick *only* one of the following" or "Please tick *at least one of the following"?

Reply:
@geky @adbridge ^^^

Reply:
Only sounds fine. It should be one of these , not multiple selection here (if it is, PR should be split, doing multiple jobs).

Reply:
I think the wording there is fine, please tick one , means just tick one :)